apps/nextjs-website/src/config.ts

```diff
@@ -1,4 +1,14 @@
 // TODO: Add environment parser
+
+const convertToNumberOrUndefined = (
+  value: string | undefined
+): number | undefined => {
+  if (value) {
+    return parseInt(value, 10);
+  }
+  return undefined;
+};
+
```

Comment:
I didn't omit the base `10` because of this phrase in [MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#radix):
`If 0 or not provided, the radix will be inferred based on string's value. Be careful — this does not always default to 10!`